src/site/content/en/blog/vitals-field-measurement-best-practices/index.md

```diff
@@ -73,14 +73,35 @@ function sendToAnalytics({name, value, id}) {
   const body = JSON.stringify({name, value, id});
   // Use `navigator.sendBeacon()` if available, falling back to `fetch()`.
   (navigator.sendBeacon && navigator.sendBeacon('/analytics', body)) ||
-      fetch('/analytics', {body, method: 'POST', keepalive: true});
+    fetch('/analytics', {body, method: 'POST', keepalive: true});
```

Comment:
The 4-space indent here is actually per Google JS style. Any line continuation that's not a block indentation (like a object or array literal) should be indented 4 spaces instead of 2.

Reply:
I figure Prettier must have done something here. My editor's been kind of goofy lately with picking up config files for stuff like eslint or prettier.